examples/generative/corrdiff/conf/config_train_base.yaml

```diff
@@ -40,8 +40,9 @@ augment: 0.0
   # Augment probability
 
 ## Performance options
-fp16: false
-  # Enable mixed-precision training
+fp_mode: fp32
```

Comment:
Would prefer renaming this to `tensor_core_mode` or `matmul_fp_mode` since this option should not affect group norm and other precision sensitive non-matmul layers.

Reply:
I think if we don't want to specifically indicate tensor cores or matmuls, we can maybe rename to `fp_compute_mode`? That would cover TF32 as well. If that's not acceptible, `fp_optimizations` is okay.

Reply:
Hmm, I think `fp_compute_mode` would be too general in the same way `fp_mode` is. Renamed to `fp_optimizations` in https://github.com/NVIDIA/modulus/pull/423/commits/de4f91833da024d35098009f09cca8a1808a2784.